.github/ISSUE_TEMPLATE/enhancement_report.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Enhancement report
+about: Create an enhancement report
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+
+**Type: [Code Style, Refactor, Documentation, Others]**
+
+**Description:**
+
```

Comment:
The bug report template doesn't haver these lines between the category and the placeholder text, we should standardize this across all templates.